components/modal.css

```diff
@@ -0,0 +1,27 @@
+.modal.fade {
+  z-index: 3;
+  display: none;
+  padding-top: 100px;
```

Comment:
relative units could be used here.

Reply:
thanks for pointing it out, will update it